scripts/error-codes/codes.json

```diff
@@ -221,7 +221,7 @@
   "219": "getInspectorDataForViewTag() is not available in production",
   "220": "Container does not support insertBefore operation",
   "221": "Tried to register two views with the same name %s",
-  "222": "View config not found for name %s",
+  "222": "View config not found for %s. If you made a component starting with a lowercase letter, make it uppercase.",
```

Comment:
Please don't edit this file. It's generated automatically on releases. Editing it now means people on older versions won't be able to find this error anymore.